implementations/micrometer-registry-new-relic/src/main/java/io/micrometer/newrelic/NewRelicInsightsAgentClientProvider.java

```diff
@@ -252,4 +258,8 @@ void sendEvents(Meter.Id id, Map<String, Object> attributes) {
             }
         }
     }
+    
+    public void setNamingConvention(NamingConvention namingConvention) {
```

Comment:
Actually, I'm wondering if this should even be public. To simplify things, we want it to all go through the builder/registry, but this allows things to be changed outside of those.

Reply:
Of course methods inherited from an interface can't be package-private, so that complicates things...